### PR TITLE
Adding faq entry: How to teach Bayes ham and spam #1438

### LINKED ISSUE #1438: How to teach Bayes ham and spam
The requirement for 200 ham for Bayes classification is not easy to meet with low-volume servers. rspamd works reasonably for the usual viagnigeria spam, but it fails completely for the dozens of requests per day to submit my publication to some fake-open-source journal

I am used to use sa-learn with Spamassassin or rspamc with rspamd to use existing mail for ham education,  but ther is no rspamc. The spam/learn tab in rspamd is ridiculous - should I manually paste in hundreds of ham messages?

What is the best method to instruct a mailu installation to use an existing clean mailbox as ham-fodder?

### PR DESCRIPTION
**Pull request by ofthesun9**
Fix #1438

## What type of PR?
documentation (faq)

## What does this PR do?
This PR add an faq entry to cover #1438 

### Related issue(s)
closes #1438 

## Prerequistes
- [x] In case of feature or enhancement: documentation updated accordingly

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -532,6 +532,26 @@ In any case, using a dedicated DNS server will improve the performance of your m
...
+```````````````````````````````````````````````````````````````
+Mailu is supporting automatic spam learning for messages moved to the Junk mailbox. Any email moved from the Junk Folder will learnt as ham. 
+
+If you already have an existing mailbox and want Mailu to learn them all as ham messages, you might run rspamc from within the dovecot container:
+
+.. code-block:: bash
+
+  rspamc -h antispam:11334 -P mailu -f 13 fuzzy_add /mail/user\@example.com/.Ham_Learn/cur/  `
+
+This should learn every file located in the Ham_Learn mailbox from user@example.com
```
Comment (muhlemmer) [inline docs/faq.rst:545]:
```suggestion
This should learn every file located in the ``Ham_Learn`` folder from user@example.com 
```

File: docs/faq.rst [hunk 2]
```diff
@@ -532,6 +532,26 @@ In any case, using a dedicated DNS server will improve the performance of your m
...
 
+Can I learn ham/spam messages from an already existing mailbox?
+```````````````````````````````````````````````````````````````
+Mailu is supporting automatic spam learning for messages moved to the Junk mailbox. Any email moved from the Junk Folder will learnt as ham. 
+
+If you already have an existing mailbox and want Mailu to learn them all as ham messages, you might run rspamc from within the dovecot container:
+
+.. code-block:: bash
+
+  rspamc -h antispam:11334 -P mailu -f 13 fuzzy_add /mail/user\@example.com/.Ham_Learn/cur/  `
```
Comment (muhlemmer) [inline docs/faq.rst:543]:
```suggestion
  rspamc -h antispam:11334 -P mailu -f 13 fuzzy_add /mail/user\@example.com/.Ham_Learn/cur/
```

File: docs/faq.rst [hunk 3]
```diff
@@ -532,6 +532,26 @@ In any case, using a dedicated DNS server will improve the performance of your m
...
+
+This should learn every file located in the Ham_Learn mailbox from user@example.com 
+
+Likewise, to lean all messages within the folder Spam_Learn as spam messages :
```
Comment (muhlemmer) [inline docs/faq.rst:547]:
```suggestion
Likewise, to lean all messages within the folder ``Spam_Learn`` as spam messages :
```

**Comment by ofthesun9** [comment]
bors try

**Pull request merged**